lib/perl/Genome/Model/Build/Command/Start.pm

```diff
@@ -160,7 +160,7 @@ sub create_and_start_build {
 
     my $start_transaction = UR::Context::Transaction->begin();
     my $build_started = try {
-        $build->start(%{$self->_start_params});
+        $build->start(%{$self->_start_params}) || die "Couldn't start build";
```

Comment:
`$build->start` currently does its own `eval` and sets its status to `Unstartable` if anything goes wrong.  Adding a `die` here will abort that process and remove the note that explains what happened.